patches/minecraft/net/minecraft/client/Minecraft.java.patch

```diff
@@ -29,7 +29,7 @@
        }
  
        Util.f_137440_ = RenderSystem.m_69576_();
-+      if (net.minecraftforge.common.ForgeConfig.CLIENT.forceSystemNanoTime.get()) Util.f_137440_ = System::nanoTime;
++      if (net.minecraftforge.common.ForgeConfig.CLIENT.forceSystemNanoTime.getRaw()) Util.f_137440_ = System::nanoTime;
```

Comment:
If this runs before configs are loaded, doesn't that mean that the option does not do anything? This code only runs once during startup, the usual usecase for `getRaw` should be code that is called both before and after the config is loaded.

Thanks for implementing this btw!

Reply:
Yes; as currently implemented, the `forceSystemNanoTime` option on the Forge client config does not affect anything at all. By this reasoning, the config option should be removed (or possibly moved to a later position), but I'm presuming that change would be out of this PR's scope, and am leaving it in for now for a later PR to remove it. 

If wanted, I can mark the option for removal in a later PR with a `// TODO` comment, or fix/remove the option in this PR, though I'd like to keep the scope of this PR as-is and leave the matter of fixing/removing it for someone else more knowledgable on its effects (or lack thereof) on rendering).

Reply:
This is actually the perfect example for the use case of this PR - the current `get` method may cause cases where the config isn't respected without the modder recognising this. I would at least add a `TODO` comment here, but if you want to fix it directly in this PR I personally wouldn't consider this Out of Scope